frontend/tests/unit/tests/work_packages/activities/revision-activity-directive-test.js

```diff
@@ -0,0 +1,139 @@
+//-- copyright
+// OpenProject is a project management system.
+// Copyright (C) 2012-2015 the OpenProject Foundation (OPF)
+//
+// This program is free software; you can redistribute it and/or
+// modify it under the terms of the GNU General Public License version 3.
+//
+// OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
+// Copyright (C) 2006-2013 Jean-Philippe Lang
+// Copyright (C) 2010-2013 the ChiliProject Team
+//
+// This program is free software; you can redistribute it and/or
+// modify it under the terms of the GNU General Public License
+// as published by the Free Software Foundation; either version 2
+// of the License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program; if not, write to the Free Software
+// Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+//
+// See doc/COPYRIGHT.rdoc for more details.
+//++
+
+/*jshint expr: true*/
+
+describe('revisionActivity Directive', function() {
+    var compile, element, rootScope, scope;
+
+    beforeEach(angular.mock.module('openproject.workPackages.activities'));
+    beforeEach(function() {
+      module(
+        'ng-context-menu',
+        'openproject.api',
+        'openproject.workPackages',
+        'openproject.models',
+        'openproject.services',
+        'openproject.config',
+        'openproject.templates'
+      );
+    });
+
+    beforeEach(inject(function($rootScope, $compile, $uiViewScroll, $timeout, $location, I18n, PathHelper, ActivityService, UsersHelper) {
```

Comment:
Line is too long.<br>'UsersHelper' is defined but never used.<br>'ActivityService' is defined but never used.<br>'PathHelper' is defined but never used.<br>'I18n' is defined but never used.<br>'$location' is defined but never used.<br>'$timeout' is defined but never used.<br>'$uiViewScroll' is defined but never used.